src/app/submission/components/submit-to-scea/submit-to-scea.component.ts

```diff
@@ -0,0 +1,104 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Project} from "@shared/models/project";
+import {AlertService} from "@shared/services/alert.service";
+import {BrokerService} from "@shared/services/broker.service";
+import {LoaderService} from "@shared/services/loader.service";
+import {SaveFileService} from "@shared/services/save-file.service";
+import {Observable} from "rxjs";
+import {FormGroup, FormControl, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-submit-to-scea',
+  templateUrl: './submit-to-scea.component.html',
+  styleUrls: ['./submit-to-scea.component.css']
+})
+
+export class SubmitToSCEAComponent implements OnInit {
+
+  @Input() project$: Observable<Project>;
+  @Input() submissionEnvelopeId;
+  @Input() submissionEnvelope$;
+  @Input() isSubmitted: boolean;
+  @Input() submissionUrl: string;
+  @Input() isLinkingDone: boolean;
+  @Input() manifest: object;
+  releaseDate: string;
```

Comment:
All of these properties are necessary. You can access them all directly from the FormGroup

Reply:
Did you mean **un**necessary?